storage/connect/reldef.cpp

```diff
@@ -533,7 +575,9 @@ PTABDEF OEMDEF::GetXdef(PGLOBAL g)
 #endif // 0
 
   // Is the library already loaded?
+#if !defined(_AIX)
   if (!Hdll && !(Hdll = dlopen(soname, RTLD_NOLOAD)))
+#endif
```

Comment:
ok i confused it with the other change in this file.

this is already undefined because this code makes no sense in aix.

  // Is the library already loaded?
#if !defined(_AIX)
  if (!Hdll && !(Hdll = dlopen(soname, RTLD_NOLOAD)))
#endif